app/navigation/IndividualBeneficiaryNavigator.scala

```diff
@@ -35,31 +35,166 @@ class IndividualBeneficiaryNavigator @Inject()() extends Navigator {
+        yesCall = navigateAwayFromCountryOfResidencyQuestions(ua, mode),
+        noCall = rts.CountryOfResidenceController.onPageLoad(mode)
+      )
     case AddressYesNoPage => ua =>
-      yesNoNav(ua, AddressYesNoPage, rts.LiveInTheUkYesNoController.onPageLoad(mode), rts.VPE1FormYesNoController.onPageLoad(mode))
+      yesNoNav(
+        ua = ua,
+        fromPage = AddressYesNoPage,
+        yesCall = rts.LiveInTheUkYesNoController.onPageLoad(mode),
+        noCall = navigateToMentalCapacityOrVulnerableQuestions(ua, mode)
+      )
     case LiveInTheUkYesNoPage => ua =>
         yesNoNav(ua, LiveInTheUkYesNoPage, rts.UkAddressController.onPageLoad(mode), rts.NonUkAddressController.onPageLoad(mode))
     case PassportDetailsYesNoPage => ua =>
         yesNoNav(ua, PassportDetailsYesNoPage, addRts.PassportDetailsController.onPageLoad(), addRts.IdCardDetailsYesNoController.onPageLoad())
     case IdCardDetailsYesNoPage => ua =>
-        yesNoNav(ua, IdCardDetailsYesNoPage, addRts.IdCardDetailsController.onPageLoad(), rts.VPE1FormYesNoController.onPageLoad(mode))
+      yesNoNav(
+        ua = ua,
+        fromPage = IdCardDetailsYesNoPage,
+        yesCall = addRts.IdCardDetailsController.onPageLoad(),
+        noCall = navigateToMentalCapacityOrVulnerableQuestions(ua, mode)
+      )
     case PassportOrIdCardDetailsYesNoPage => ua =>
-      yesNoNav(ua, PassportOrIdCardDetailsYesNoPage, amendRts.PassportOrIdCardDetailsController.onPageLoad(), rts.VPE1FormYesNoController.onPageLoad(mode))
+      yesNoNav(
+        ua = ua,
+        fromPage = PassportOrIdCardDetailsYesNoPage,
+        yesCall = amendRts.PassportOrIdCardDetailsController.onPageLoad(),
+        noCall = navigateToMentalCapacityOrVulnerableQuestions(ua, mode)
+      )
+  }
+
+  private def navigateAwayFromDateOfBirthPages(ua: UserAnswers, mode: Mode): Call = {
+    if (ua.isTaxable) {
+      rts.IncomeDiscretionYesNoController.onPageLoad(mode)
+    } else {
+      rts.CountryOfNationalityYesNoController.onPageLoad(mode)
+    }
+  }
+
+  private def navigateAwayFromShareOfIncomeQuestions(ua: UserAnswers, mode: Mode): Call = {
+    if (ua.is5mldEnabled) {
+      rts.CountryOfNationalityYesNoController.onPageLoad(mode)
+    } else {
+      rts.NationalInsuranceNumberYesNoController.onPageLoad(mode)
+    }
+  }
+
+  private def navigateAwayFromCountryOfNationalityQuestions(ua: UserAnswers, mode: Mode): Call = {
+    if (ua.isTaxable) {
+      rts.NationalInsuranceNumberYesNoController.onPageLoad(mode)
+    } else {
+      rts.CountryOfResidenceYesNoController.onPageLoad(mode)
+    }
+  }
+
+  private def navigateAwayFromCountryOfResidencyQuestions(ua: UserAnswers, mode: Mode): Call = {
+    (ua.get(NationalInsuranceNumberYesNoPage), ua.isTaxable) match {
+      case (Some(true), _) => rts.MentalCapacityYesNoController.onPageLoad(mode)
+      case (_, true) => rts.AddressYesNoController.onPageLoad(mode)
+      case _ => rts.MentalCapacityYesNoController.onPageLoad(mode)
```

Comment:
Could this be done in 2 cases? Perhaps
`case (Some(true), _) | (_, false) => rts.MentalCapacityYesNoController.onPageLoad(mode)`
`case _ => rts.AddressYesNoController.onPageLoad(mode)`

Reply:
Updated